tools/Get-NuGetTool.ps1

```diff
@@ -0,0 +1,15 @@
+<#
+.SYNOPSIS
+    Downloads the NuGet.exe tool and returns the path to it.
+#>
+
+$binaryToolsPath = "$PSScriptRoot\..\obj\tools"
+if (!(Test-Path $binaryToolsPath)) { $null = mkdir $binaryToolsPath }
+$nugetPath = "$binaryToolsPath\nuget.exe"
+if (!(Test-Path $nugetPath)) {
+    $NuGetVersion = "3.4.4"
```

Comment:
Is there a way to just get the latest or are we afraid for breaking changes when we always get the latest?